README.md

```diff
@@ -1,3 +1,5 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela/selene-ah-backend/badge.svg?branch=master)](https://coveralls.io/github/andela/selene-ah-backend?branch=develop)
```

Comment:
@JohnMadakin the target branch should be `develop`

Reply:
ok, will resolve it now